tools/component-builder/css/lib/varUtils.js

```diff
@@ -169,7 +169,9 @@ function getAllVars() {
 				`${varDir}/css/globals/*.css`,
 				`${varDir}/custom.css`,
 				coreTokensFile,
-			])
+			], {
+				allowEmpty: true,
```

Comment:
Allows us to delete empty themes assets from components that don't need express styles